march_gain_scheduling/src/march_gain_scheduling/one_step_linear_interpolation.py

```
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+import rospy
+
+
+def interpolate(current_gains, needed_gains, gradient, delta_t):
+    if len(current_gains) != len(needed_gains):
+        rospy.logerr("needed_gains and current_gains not equal length")
+        return current_gains
```

Throw an exception here instead of returning incorrect values